app/assets/javascripts/oxalis/model/actions/view_mode_actions.js

```diff
@@ -1,18 +1,76 @@
 // @flow
 
+import constants from "oxalis/constants";
 import type { OrthoViewType } from "oxalis/constants";
+import type { PartialCameraData } from "oxalis/store";
+import * as THREE from "three";
+import { getTDViewportSize } from "oxalis/model/accessors/view_mode_accessor";
 
 type SetViewportActionType = {
   type: "SET_VIEWPORT",
   viewport: OrthoViewType,
 };
 
+type SetTDCameraActionType = {
+	type: "SET_TD_CAMERA",
```

Comment:
Looks like you mixed tabs and spaces here. We're using spaces everywhere.
This won't happen again if you install the EditorConfig package for Sublime or Atom as we're having a .editorconfig file in the repo :)